scripts/generate_data/index.js

```diff
@@ -0,0 +1,85 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+const { program, InvalidArgumentError } = require('commander');
+const { createWriteStream } = require('fs');
+const { join } = require('path');
+const { BasicDocumentGenerator } = require('./basic_generator');
+
+const DEFAULT_PATH = './cypress/fixtures/dashboard/opensearch_dashboards/';
+const DOCUMENT_GENERATORS = {
+  basic: BasicDocumentGenerator,
+};
+
+const defaultGenerator = Object.keys(DOCUMENT_GENERATORS)[0];
+
+program
+  .argument('{index}', 'Index for the test data')
+  .argument('[path]', 'path to the test data')
+  .option(
+    '-n, --doc-count [number]',
+    'Number of documents to generate',
+    toInt,
+    '10000'
+  )
+  .option(
+    '-g, --generator [string]',
+    `Available Generators: ${Object.keys(DOCUMENT_GENERATORS).join(',')}`,
```

Comment:
nice, could we also add basic usage to developer guide if not yet.

```
node scripts\generate_data xxx xxx
or
npm generate:test-data
```

Reply:
I kept it out of there for now because I wanted to have a lil more adoption from other plugins to make sure that this script is general enough for other use cases.